text/0000-adoption-of-a-rustacean-crustacen.md

```diff
@@ -0,0 +1,86 @@
+- Feature Name: Adopt Ferris as the Official Mascot/Epoch Costumes
+- Start Date: 2018-02-01
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Officially adopt Ferris as the mascot for the current epoch and have a new
+costume/design for each epoch as it arrives.
+
+# Motivation
+[motivation]: #motivation
+
+Ferris has been a part of the community for many years and used as an unofficial
+mascot for Rust. Our nickname Rustacean is a reference to Ferris the Crustacean.
```

Comment:
Ferris is Ferris the Rustacean, not Ferris the Crustacean. “Our nickname Rustacean came from Ferris the Rustacean (being a play on *crustacean*).” would work.